assets/src/components/imageUpload/style.scss

```diff
@@ -0,0 +1,40 @@
+/**
+ * Image uploader styles.
+ */
+
+.newspack-image-upload {
+	button {
+		color: $muriel-hot-pink-500;
+		font-size: 16px;
+		padding: 0;
+		vertical-align: middle;
+	}
+
+	.image-preview {
+		font-size: 16px;
+		margin-right: 2em;
+
+		img {
+			max-width: 240px;
+			height: auto;
+		}
+	}
+
+	&.no-image {
+		.add-image {
+			&:before {
```

Comment:
This is nice. Makes me think we should revisit the [`infoButton` styling](https://github.com/Automattic/newspack-plugin/blob/master/assets/src/components/infoButton/style.scss#L5-L12) as a `:before` to unify our approach?

Reply:
Yeah that sounds like a good idea. It keeps the markup and the style separate. I'll just add that small tweak to the revisions for this PR.

Reply:
I gave it a shot. Tooltip needs to wrap an element, (i.e. the thing you hover over over to display the tooltip), so we still need the `newspack-info-button` div. Since we still need that div, there isn't really any benefit to changing the style to use `:before`.